migration/mysql/20240822110001_update_push_table.sql

```diff
@@ -0,0 +1,2 @@
+-- Modify "push" table
+ALTER TABLE `push` MODIFY COLUMN `fcm_api_key` varchar(511) NULL;
```

Comment:
I'll drop the column once the new API is released. So, for now, I'm making it optional so it won't fail when creating new pushes.